src/policy/descriptor.rs

```diff
@@ -276,29 +277,50 @@ impl Descriptor<DescriptorPublicKey> {
 
 impl<Pk: MiniscriptKey> fmt::Display for Descriptor<Pk> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        fmt::Display::fmt(&self.policy, f)
+        write!(f, "sim({},{})", self.internal_key, self.policy)
```

Comment:
In afae99a2e01351a1b8d1eb7b93802e05b0501498:

This commit seems to introduce a new `sim` descriptor. What is the value of having such a descriptor? What is the difference between a `sim` and a normal `tr` descriptor?

Reply:
`sim` takes a Simplicity policy as second argument while `tr` takes Miniscript. We can overload `tr` if you want, but that might be confusing.

Reply:
I'm confused. The difference between Simplicity and Script happens at the tapleaf level. It wouldn't be "overloading" `tr` to use it for different kinds of taproot descriptors.